content/docs/user-guide/interactivity/robotics/creating-robotic-simulation.md

```diff
@@ -0,0 +1,36 @@
+---
+linkTitle: Creating a Robotic Simulation
+title: Creating a Robotic Simulation
+description: A step-by-step process on how to create a robotic simulation using the ROS 2 Gem in Open 3D Engine (O3DE).
+toc: true
+---
+
+## How to create your own robotic simulation
+
+### Tutorials
+
+>This section is to be filled in
+
+### High-level steps
+
+>This section is to be detailed.
+
```

Comment:
Because we're approaching a release in May, I suggest commenting out any future or in-progress sections like these.